aries-site/src/examples/templates/dashboards/components/UpdatesFeed.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import { Card, CardBody, CardHeader, ThemeContext } from 'grommet';
+import { useContext } from 'react';
+import { DashboardCardHeader } from '.';
+
+export const UpdatesFeed = ({ background, children, menuItems, title }) => {
+  const theme = useContext(ThemeContext);
+  const { body, header } = theme.card;
+
+  return (
+    <Card background={background || 'background'}>
```

Comment:
One other implementation detail which is a little tricky is that the `validation-warning` color has opacity to it. This works fine when it sits atop the intended background ('background').

So, I think we may need to do something like:
```
<Card>
  <Box background={background}>
    <CardHeader>
      ...
```

Reply:
good catch I didnt realize the yellow was darker I but a `box` and yes you can visually see the correct color